03week/towersOfHanoi.js

```diff
@@ -13,29 +13,68 @@ let stacks = {
   c: []
 };
 
+/*
+- a, b, c are the stacks/columns that the blocks go on
+- the numbers 1, 2, 3, 4 are the blocks, with 1 being the smallest block and 4 being the largest
+- use .push to add something to the end
+- use .pop to remove what's on the end of the stack
+- check to see if the last number on the stack is smaller than the numbers before it
+*/
+
 function printStacks() {
   console.log("a: " + stacks.a);
   console.log("b: " + stacks.b);
   console.log("c: " + stacks.c);
 }
 
 function movePiece() {
-  // Your code here
-
+  // when one stack is clicked, remove the last block on it
+  // add that removed block to the next column the user clicks
+  let strClickedPiece = stacks.a.pop();
+  if (stacks.b[stacks.b.length - 1] > strClickedPiece) { // to check if the last block present is bigger than the one being appended
```

Comment:
this code is redundant, you're already doing this in the isLegal(), also the only thing this function should be doing is moving the piece.